website/src/pages/remote-cache.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import Layout from "@theme/Layout";
+import common from "../css/common.module.css";
+
+import Hero from "../components/hero/hero";
+import CTA from "../components/cta/cta";
+import Globe from "../components/globe/globe";
+
+function Component() {
+  return (
+    <Layout title="Bazel at Enterprise Scale">
+      <div className={common.page}>
+        <Hero
+          title="Remote Cache"
+          subtitle="Global remote caching infrastructure made easy. Highly scalable, blazing fast, and incredibly simple to setup."
+          component={<Globe />}
+          lessPadding={true}
+          flipped={true}
+        />
+        <Hero
+          title="Globally distributed"
+          subtitle="BuildBuddy Cloud cache has 61 points of presence around the world to minimize ping and maximize throughput."
+          image={require("../../static/img/ping.png")}
+          bigImage={true}
+        />
+        <Hero
+          title="Cache stats"
+          subtitle="Tools to track and monitor your cache hit ratios over time. Catch regressions and "
```

Comment:
catch regressions and.....?

duh duh duh so curious

Reply:
Fixed